src/App.tsx

```diff
@@ -1,9 +1,58 @@
 import './App.scss';
+import { useEffect, useState } from 'react';
+import { NavLink, Route, Routes } from 'react-router-dom';
+import 'bulma/css/bulma.css';
 
-const App = () => (
-  <div className="App">
-    <h1>People table</h1>
-  </div>
-);
+import { getPeople } from './api';
+import { PeopleTable } from './components/PeopleTable/PeopleTable';
+import { Loader } from './components/Loader';
 
-export default App;
+export const App:React.FC = () => {
+  const [people, setPeople] = useState<Person[]>([]);
+
+  const activeLink = ({ isActive }: any) => (isActive
+    ? 'title is-1 has-text-primary'
+    : 'title is-1 has-text-white');
+
+  useEffect(() => {
+    getPeople().then(data => setPeople(data));
+  }, []);
+
+  return (
+    <div className="App">
+      <header className="App__header ">
+        <nav className="App__nav">
+          <NavLink
+            className={activeLink}
+            to="/"
+          >
+            HomePage
+          </NavLink>
+          <NavLink
+            className={activeLink}
+            to="/people"
+          >
+            PeoplePage
+          </NavLink>
```

Comment:
you should also implement this part:
"5. Redirect to / from /home (use [Navigate](https://reactrouter.com/docs/en/v6/components/navigate))"